src/redux/actions/profile/fetchProfile.test.js

```diff
@@ -0,0 +1,21 @@
+import { fetchProfile, updateProfile } from './fetchProfile';
+import mockFetch from '../../../__mocks__/mockFetch';
+
+const dispatch = jest.fn(() => ({ data: {} }));
+describe('profile calling backend services', () => {
+  it('should fetch profile data from authors haven API', async () => {
+    window.fetch = mockFetch;
+    fetchProfile()(dispatch);
+    expect(true).toBe(true);
+  });
+  it('should update profile', async () => {
+    window.fetch = mockFetch;
+    const data = {
+      bio: 'this is a sample bio',
+      image: 'image.jpeg',
+      username: 'admin',
+    };
+    await updateProfile(data)(dispatch);
+    expect(true).toBe(true);
```

Comment:
OMG, how do you think this will fail???? expect true to be true.

Reply:
😱Accidentally left this when I was debugging. Let me fix this ASAP